RF24.h

```diff
@@ -499,6 +516,9 @@ class RF24
      * New: Open a pipe for writing via byte array. Old addressing format retained
      * for compatibility.
      *
+     * @deprecated Use `RF24::txAddress` instead.
+     * As of v1.5, using this function is slower than using `RF24::txAddress`.
```

Comment:
This patch implies a minor version bump. Network layers (after adopting `txAddress`) would only get a patch version bump.